packages/flex-dev-utils/src/fs.ts

```diff
@@ -45,16 +46,30 @@ export const checkFilesExist = (...files: string[]) => {
  */
 export const getPackageJsonPath = (forModule: boolean = false) => path.join(process.cwd(), 'package.json');
 
+/**
+ * Gets package-lock.json path
+ */
+export const getPackageLockJsonPath = (forModule: boolean = false) => path.join(process.cwd(), 'package-lock.json');
```

Comment:
Our `getPackageJsonPath` has a `forModule` var that is not being used (I'll remove it). Remove this variable here too as it is unused